lib/jxl/jxl_test.cc

```diff
@@ -863,7 +863,7 @@ TEST(JxlTest, RoundtripAlphaResamplingOnlyAlpha) {
   CodecInOut io2;
   EXPECT_TRUE(DecodeFile(dparams, compressed, &io2, pool));
 
-  EXPECT_LE(compressed.size(), 34000u);
+  EXPECT_LE(compressed.size(), 34020u);
```

Comment:
Can we be a bit less tight? :D

Reply:
Made it a bit less tight, but I would prefer to keep the margin small so we know when density regressions are happening.